client/layout/masterbar/style.scss

```diff
@@ -557,12 +557,19 @@ body.is-mobile-app-view {
 			display: block;
 		}
 
-		&.masterbar__item-notifications,
-		&.masterbar__item-help {
+		&.masterbar__item-my-site-actions {
+			display: none;
+		}
+	}
+
+	@media only screen and (max-width: 400px) {
+		&.masterbar__item-my-site-actions,
+		&.masterbar__reader {
 			display: none;
 		}
 	}
 
+
```

Comment:
Looks like you may have added an extra line here?

Reply:
Fixed :wink: